openpdf/src/main/java/com/lowagie/text/pdf/PdfContentByte.java

```diff
@@ -3043,7 +3060,18 @@ else if (obj.isArray()) {
         }
         pdf.incMarkPoint();
         mcDepth++;
-        content.append(struc.get(PdfName.S).getBytes()).append(" <</MCID ").append(mark).append(">> BDC").append_i(separator);
+        // TIBCO Software #4 : Part 1 - START
+        //content.append(struc.get(PdfName.S).getBytes()).append(" <</MCID ").append(mark).append(">> BDC").append_i(separator);
```

Comment:
No point in keeping commented out code if it is replaced with better code, unless it is to explain better what is going on in the new code.